core/templates/dev/head/pages/topics_and_skills_dashboard/SkillsListDirective.js

```diff
@@ -177,30 +175,10 @@ oppia.directive('skillsList', [
             modalInstance.result.then(function(result) {
               var skill = result.skill;
               var supersedingSkillId = result.supersedingSkillId;
-              var changeList = [{
-                cmd: 'update_skill_property',
-                property_name: 'superseding_skill_id',
-                old_value: '',
-                new_value: supersedingSkillId
-              }];
-              EditableSkillBackendApiService.updateSkill(
```

Comment:
There's a HTTP request underneath this one to editableSkillDataUrl. Just to double-check, is it correct that this is no longer needed?

Also just to check, do we have e2e tests for this flow?

Reply:
The same functionality done by the HTTP request is done in `MergeSkillHandler` after the questions are successfully transferred, and hence that request is not necessary.
No, I don't think e2e tests were written for the merge skill path. Should I include it in this PR?

Reply:
Thanks for the explanation :)

And, for e2e tests -- I'd say yes, since that path broke in a production server!

Reply:
Yeah, will add it.

Reply:
Done